core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -978,21 +970,19 @@ public OperationHandler<Operation> getOperationHandler(final Class<? extends Ope
         return operationHandlers.get(opClass);
     }
 
-    private JobDetail addOrUpdateJobDetail(final OperationChain<?> operationChain, final Context context,
-                                           final String msg, final JobStatus jobStatus) {
+    private JobDetail addOrUpdateJobDetail(final OperationChain<?> operationChain, final Context context, final String msg, final JobStatus jobStatus) {
         final JobDetail newJobDetail = new JobDetail(context.getJobId(), context.getUser(), operationChain, jobStatus, msg);
-        if (nonNull(jobTracker)) {
+        if (null != jobTracker) {
```

Comment:
nitpick but we're transitioning away from yoda conditions so ideally the changes to this method would be reverted (which will also fix the code coverage issues), or reverse the conditions

Reply:
reverted :)